resources/assets/enderio/config/SAGMillRecipes_Core.xml

```diff
@@ -686,22 +722,21 @@ addOreDictionaryTooltips=true
   </recipeGroup>
 
 
-  <recipeGroup name="Applied Energistics">
-
+  <recipeGroup name="Applied Energistics 2">
 
     <recipe name="ChargedCertusQuartzOre" energyCost="2400">
       <input>
-        <itemStack  modID="appliedenergistics2" itemName="tile.OreQuartzCharged"  />
+        <itemStack modID="appliedenergistics2" itemName="tile.OreQuartzCharged" />
       </input>
       <output>
-        <itemStack modID="appliedenergistics2" itemName="item.ItemMultiMaterial" itemMeta="1"/>
+        <itemStack modID="appliedenergistics2" itemName="item.ItemMultiMaterial" itemMeta="1" />
```

Comment:
One ore = one crystal and one dust. Isn't this right? Usually one ore = two dusts but if there are will be two crystals, the full output will be 3 dusts (since crystals can be crushed down to dusts)